codec-http/src/test/java/io/netty/handler/codec/http/cookie/CookieDateFormatterTest.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2016 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.handler.codec.http.cookie;
+
+import org.junit.Test;
+
+import java.util.Date;
+
+import static org.junit.Assert.assertEquals;
+
+public class CookieDateFormatterTest {
+    /**
+     * This date is set at "06 Nov 1994 08:49:37 GMT" (same used in example in
+     * RFC documentation)
```

Comment:
Doh, I'll have to dig. I forked existing `io.netty.handler.codec.http.HttpHeaderDateFormatTest`.